packages/sdk/test/evm/verification.test.ts

```diff
@@ -0,0 +1,44 @@
+import dotenv from "dotenv";
+import {
+  isVerifiedOnEtherscan,
+  verify,
+} from "../../src/evm/common/verification";
+import { assert } from "chai";
+import { ThirdwebStorage } from "@thirdweb-dev/storage";
+
+dotenv.config();
+
+describe("Contract Verification", async () => {
+  const api = "https://api-goerli.etherscan.io/api";
+  const apiKey = process.env.API_KEY as string;
```

Comment:
Probably don't want to have unit tests in here for this. Maybe in e2e tests. Fine to have in the code for manual tests but i would skip those before merging